### PR TITLE
MM-29607 allow user to deactivate MFA even when disabled

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost-server/pull/16994

Updated:
- Created new common topic called "Disabling MFA" to be included in both the Self-Managed Admin Guide and the Cloud Admin Guide topics on using Multi-Factor Authentication

### CODE REVIEW

File: source/cloud/cloud-administration/common-disable-mfa.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Disabling MFA
+=============
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/common-disable-mfa.rst:2]:
```suggestion
-------------
```

File: source/cloud/cloud-administration/common-disable-mfa.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Disabling MFA
+-------------
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/common-disable-mfa.rst:2]:
```suggestion
^^^^^^^^^^^^^
```

File: source/cloud/cloud-administration/common-disable-mfa.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+System Admins can disable this option by going to **System Console > Authentication > MFA**. When **Enable Multi-factor Authentication** is set to **false**, users can't opt to use or disable MFA on their account in  **Account Settings > Security > Multi-factor Authentication**.
+
+MFA can be disabled for user accounts using the API. See the `Mattermost API Reference <https://api.mattermost.com/#tag/users/paths/~1users~1{user_id}~1mfa/put>'__ for details.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/common-disable-mfa.rst:6]:
```suggestion
MFA can be disabled for user accounts using the API. See the `Mattermost API Reference <https://api.mattermost.com/#tag/users/paths/~1users~1{user_id}~1mfa/put>`__ for details.
```

**Review by justinegeffen — APPROVED** [review]
This is very cool, love the common file! :)

**Review by adamjclarkson — APPROVED** [review]

**Review by sbishel — APPROVED** [review]

**Pull request merged**